fastlane/metadata/android/en-US/full_description.txt

```diff
@@ -0,0 +1,6 @@
+ownCloud is a free software package you can install on a server to manage files,
```

Comment:
Why do you have all these newlines here? IMO it's better to let the end user program decide when to make a newline.

Reply:
This description is what will appear in the ownCloud tab in F-Droid, I don't get this comment, sorry

Reply:
This text contains a lot of line breaks. Wouldn't it be better to remove all line breaks and let the F-Droid client and the F-Droid Website decide where to insert line breaks based on the text size and screen size?

Reply:
![Screenshot_20190314-172249](https://user-images.githubusercontent.com/22525368/54373641-ea767980-467d-11e9-8afd-4622ebe9cae6.png)
@davigonz Please see my screenshot. With all the needlessly newlines it looks awful.

Reply:
Understood, thanks for the screenshot, more clear now, would you like to open a PR to fix it? Thanks